components/org.wso2.transport.jms/pom.xml

```diff
@@ -18,15 +18,15 @@
   -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>org.wso2.carbon.transport.jms.parent</artifactId>
-        <groupId>org.wso2.carbon.transport</groupId>
+        <groupId>org.wso2.transport.jms</groupId>
+        <artifactId>jms-parent</artifactId>
         <version>6.0.48-SNAPSHOT</version>
         <relativePath>../../pom.xml</relativePath>
     </parent>
     <modelVersion>4.0.0</modelVersion>
     <packaging>bundle</packaging>
-    <name>WSO2 Carbon Transport JMS Component</name>
-    <artifactId>org.wso2.carbon.transport.jms</artifactId>
+    <name>WSO2 Transport JMS Component</name>
+    <artifactId>org.wso2.transport.jms</artifactId>
```

Comment:
GroupID and artefactID are the same for the component? (i.e. org.wso2.transport.jms)

Reply:
yes, as we were discussed I had to change group id to "org.wso2.transport.jms", after removing the carbon term I ended up artifact id as the same.

Reply:
what about "transport-jms" as artifact id,

Reply:
FYI : https://maven.apache.org/guides/mini/guide-naming-conventions.html

Reply:
I haven't changed that as we are still under discussion

Reply:
Updated artifact ids